deepmd/pd/model/task/invar_fitting.py

```
@@ -0,0 +1,181 @@
+    type_map: List[str], Optional
+        A list of strings. Give the name to each type of atoms.
+
+    """
+
+    def __init__(
+        self,
+        var_name: str,
+        ntypes: int,
+        dim_descrpt: int,
+        dim_out: int,
+        neuron: list[int] = [128, 128, 128],
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)